### ISSUE TITLE
Dark icons on dark background

### ISSUE THREAD

**Issue by rtakehara** [TEParsons/torillic#4]
the toggle sidebar, toggle source code mode and word count are all dark texts with a dark background, making it hard to see

I changed my torillic.css file to add the following in the end of the file

```css
#toggle-sourceview-btn,
#outline-btn,
#footer-spell-check,
#footer-word-count-label,
::before,
footer.div {
  color: #e0e5c1;
}

.ty-icon::before {
  color: inherit
}

.typora-sourceview-on #toggle-sourceview-btn {
  background: rgba(0,0,0,0);
}
.typora-sourceview-on #toggle-sourceview-btn:hover {
  background: rgba(255,255,255,0.5);
}
```

I don't know if it's the best way to implement it, but it looks better

**Comment by TEParsons** [comment]
This seems reasonable :) Are you comfortable submitting this as a Pull Request?

**Comment by rtakehara** [comment]
sure, never did such thing before but I think its about time for me to learn

**Comment by TEParsons** [comment]
Let me know if you need help going through the process :) If you add "fixes #4" to the description of the Pull Request then GitHub will detect that and automatically close this issue when it's pulled in.

**Comment by TEParsons** [comment]
Fixed in v2.0
